### ISSUE TITLE
Support for playing FLAC

### ISSUE THREAD

**Issue by toqov** [tuxfoo/jellyfin-skill#6]
Jellyfin-skill cannot play flac, only mp3 works properly.

**Comment by tuxfoo** [comment]
Is transcoding enabled for the user that you use for jellyfin-skill?
I think I have used it on some flac files, will have to check in the weekend.

**Comment by tuxfoo** [comment]
> Jellyfin-skill cannot play flac, only mp3 works properly.

I do not think this is on Jellyfins end, I just tried it with all transcoding/conversion/downloading features off and the flac songs played fine.

Could you please let me know more about your mycroft setup?

Cheers.
